src/metabase/core.clj

```diff
@@ -144,9 +142,7 @@
   ;; start scheduler at end of init!
   (task/start-scheduler!)
   (init-status/set-complete!)
-  (let [start-time (.getStartTime (ManagementFactory/getRuntimeMXBean))
-        duration   (- (System/currentTimeMillis) start-time)]
-    (log/info (trs "Metabase Initialization COMPLETE in {0}" (u/format-milliseconds duration)))))
```

Comment:
I think this worth keeping

Reply:
https://github.com/metabase/metabase/pull/31732/commits/b37629c017bd81b1696a2c784123b1765d8c753c